File: py/src/formulas/IdentityMatrix.py
def identity_matrix(n: int) -> list[list[int]]:
    res = []
    for i in range(n):
        res.append([])
        for j in range(n):
            if i == j:
                res[i].append(1)
            else:
                res[i].append(0)
    return res

File: cs/src/formulas/IdentityMatrix.cs
public static class IdentityMatrix
{
    public static int[,] Create(int n)
    {
        int[,] res = new int[n, n];
        for (int i = 0; i < n; i++)
        {
            for (int j = 0; j < n; j++)
            {
                res[i, j] = (i == j) ? 1 : 0;
            }
        }
        return res;
    }
}
